addon/styles/_frost-toggle.scss

```diff
@@ -58,9 +54,8 @@ $frost-toggle-button-hover-shadow: rgba(0, 0, 0, .2);
     &:after {
       display: block;
       position: absolute;
-      width: 40px;
+      width: 45%;
```

Comment:
Yes it's constrained by the parent element which in this case is a label with the `.frost-toggle-button` class. If the width was 100% it would span the entire width of the toggle.